configuration/mounting-paritions/en.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Automatically mounting internal partitions"
++++
+# Automatically mounting internal partitions
+
+Solus can automatically mount your internal partitions on startup.
+
+## Budgie & GNOME
```

Comment:
There needs to be dedicated sections for Budgie and GNOME, regardless of them being the same.